docker/Dockerfile

```diff
@@ -31,48 +30,28 @@ RUN yum -y install curl wget bzip2 make git patch unzip bison yasm autoconf auto
 ENV PATH /opt/rh/devtoolset-2/root/usr/bin:$PATH
 ENV LD_LIBRARY_PATH /opt/rh/devtoolset-2/root/usr/lib64:/opt/rh/devtoolset-2/root/usr/lib
 
+# Shell scripts for build
+RUN git clone https://github.com/matthew-brett/manylinux.git -b refactor-dockerfile
```

Comment:
An artefact of the possibly silly attempt to put the files in the same repo as the Dockerfile.  Alternative suggestions welcome.   Obviously, barring better suggestions, would in due course be:

```
RUN git clone https://github.com/manylinux/manylinux.git
```